### DISCUSSION TITLE
Enlarge publication preview image on click

### DISCUSSION THREAD

**Discussion by WYF99** [alshedivat/al-folio#2662] [Q&A] [answered]
My understanding of the publication preview is that it displays a small thumbnail by default, and upon click it will center and be enlarged to a more readable size, as indicated by the "zoom in" icon when putting the cursor above it.

However, in reality, the preview image is simply centered without any size change.
An example is the first publication item in the [demo page](https://alshedivat.github.io/al-folio/publications/).

Is there any way to change this behavior or is it just designed this way?

<img width="650" alt="Screenshot 2024-08-27 at 12 31 42 PM" src="https://github.com/user-attachments/assets/c698dccd-4fea-42a3-bf4d-d99d87b70ae8">
<img width="682" alt="Screenshot 2024-08-27 at 12 35 37 PM" src="https://github.com/user-attachments/assets/d6c8d6ef-c04e-41a4-b3d0-0632930ea6e4">

**Comment by george-gca** [comment, marked as answer]
In this specific case the original image is small, as you can see [in the repo](https://github.com/alshedivat/al-folio/blob/master/assets/img/publication_preview/wave-mechanics.gif), so there is nothing wrong apparently.

But by taking a look at my own site it seems that imagemagick creates a reduced version of those images, since they are displayed in this location at a really small size, and simply doesn't use the original version. Can you open an issue for this?

**Reply by WYF99** [reply]
I've opened an issue at #2668.

As a side note, thumbnails are bigger by default when accessed from my (Android) phone, although they are still not zoomed in when clicked.
